src/amo/components/AddonReviewCard/index.js

```diff
@@ -348,13 +348,10 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
     const noAuthor = shortByLine || this.isReply();
 
     if (review) {
-      const byLineString = noAuthor
-        ? // translators: Example in English: "posted last week"
-          i18n.gettext('posted %(linkStart)s%(timestamp)s%(linkEnd)s')
-        : // translators: Example in English: "by UserName123, last week"
-          i18n.gettext(
-            'by %(authorName)s, %(linkStart)s%(timestamp)s%(linkEnd)s',
-          );
+      // translators: Example in English: "posted last week"
+      const byLineString = i18n.gettext(
+        'posted %(linkStart)s%(timestamp)s%(linkEnd)s',
+      );
```

Comment:
So I think we want to keep the original `byLineString` because it is possible that translations are different in some languages (i.e. we cannot simply append both translation strings, we need the full sentence).

Reply:
Thanks for the insight! I have reverted `byLineString` back to its original format.